docs/concepts/what-is-a-list-filter.md

```diff
@@ -31,6 +31,20 @@ A List Filter contains [Search Attribute](/concepts/what-is-a-search-attribute)
   - **BETWEEN ... AND**
   - **IN**
   - **ORDER BY**
+  - **LIKE**
+
+- Wildcards ('\*', '%', etc.) are supported for string-type List Filter Search Attributes.
+
+  - Use wildcards with the **LIKE** operator to query possible values:
```

Comment:
Usually, a list (or sublist, in this instance) should have more than one list item. Make this item part of the preceding list item?

```suggestion
  Use wildcards with the **LIKE** operator to query possible values:
```

Reply:
This is what I meant as well.